### DISCUSSION TITLE
Youtube videos don't load

### DISCUSSION THREAD

**Discussion by ghost** [DandelionSprout/adfilt#442] [Q&A] [answered]
Found the problematic rule and made an exemption didn't try if other sites break, using latest version of Firefox
 $removeparam=source,domain=youtube.com
Um my bad should have posted at issues

**Comment by DandelionSprout** [comment, marked as answer]
Looking up the matter, it *seems* to have been solved in `1January2022v5-Beta`, so you can update your lists in your adblocker settings and try again.

**Comment by ghost** [comment]
Seems it's fixed thanks for the speedy answer
